src/api/spec/factories/bs_requests.rb

```diff
@@ -11,6 +11,7 @@
       target_package nil
       target_repository nil
       reviewer nil
+      review_state 'review'
```

Comment:
Why do you call this `review_state` if it's about `request.state`?

Reply:
true, will fix it